packages/fxa-auth-server/fxa-oauth-server/lib/auth_server.js

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const createBackendServiceAPI = require('../../lib/backendService');
+const Joi = require('joi');
+
+module.exports = (log, config) => {
+  const AuthServerAPI = createBackendServiceAPI(log, config, 'auth', {
+    getUserProfile: {
```

Comment:
Since all we really want is the subscription info, maybe scope this down to `getUserSubscriptions` and do not allow a scope, and only allow `subscriptions` to be returned.